contrib/playlib/src/mill/playlib/Version.scala

```diff
@@ -0,0 +1,17 @@
+package mill
+package playlib
+
+import mill.define.Target
+import mill.scalalib._
+
+private[playlib] trait Version extends Module{
+  def playVersion: T[String]
+
+  private[playlib] def playMinorVersion: T[String] = T {
+    playVersion().split("\\.").take(2).mkString(".")
+  }
+
+  private[playlib] def component(id: String): Target[Dep] = T{
```

Comment:
This should be a `T.task{}`:

    private[playlib] def component(id: String) = T.task{ ivy"com.typesafe.play::$id::${playVersion()}" }

straight `T{}` blocks can't take parameters. You just just back the first one ever called. I'm not sure how this ever worked for you. It meant that only one of the dependencies was getting pulled in.